pkg/shell/indexes.jsx

```diff
@@ -103,15 +101,9 @@ function MachinesIndex(index_options, machines, loader) {
 
     /* Host switcher enabled? */
     let host_switcher_enabled = false;
-    read_os_release().then(os_release => {
-        const enabled = os_release && get_manifest_config_matchlist(
```

Comment:
As this is gone, please also re-drop the `config:` bits from pkg/shell/manifest.json

Reply:
oh, yes! thanks!